inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp

```diff
@@ -3564,3 +3568,360 @@ INSTANTIATE_TEST_CASE_P(fusings_gpu, deconv_scale_actv_quant_u8_eltw_scale_actv_
+                            permute_params{CASE_PERMUTE_F16_3, 5, 6},
+                            permute_params{CASE_PERMUTE_F16_4, 5, 6},
+                            permute_params{CASE_PERMUTE_F16_5, 5, 6},
+                            permute_params{CASE_PERMUTE_F16_6, 5, 6},
+                            
+                            permute_params{CASE_PERMUTE_F32_3D_0, 2, 6},
+                            permute_params{CASE_PERMUTE_F32_3D_1, 2, 6},
+                            permute_params{CASE_PERMUTE_F32_3D_2, 2, 6},
+                            permute_params{CASE_PERMUTE_F32_3D_3, 2, 6},
+                            permute_params{CASE_PERMUTE_F32_3D_4, 2, 6},
+                                                                     
+                            // quantize f16 fusing not supported     
+                            permute_params{CASE_PERMUTE_F16_3D_0, 5, 6},
+                            permute_params{CASE_PERMUTE_F16_3D_1, 5, 6},
+                            permute_params{CASE_PERMUTE_F16_3D_2, 5, 6},
+                            permute_params{CASE_PERMUTE_F16_3D_3, 5, 6},
+                            permute_params{CASE_PERMUTE_F16_3D_4, 5, 6},
+                        }), );
+
+class permute_scale_actv_quant_f32_eltw_scale_actv_quant_i8: public PermuteFusingTest {};
+TEST_P(permute_scale_actv_quant_f32_eltw_scale_actv_quant_i8, basic) {
+    auto p = GetParam();
+    create_topologies(
+        input_layout("input", get_input_layout(p)),
+        data("scale1_data", get_mem(get_per_channel_layout(p), 1e-1f)),
+        data("in1_lo", get_mem(get_per_channel_layout(p), min_random, 0)),
+        data("in1_hi", get_mem(get_per_channel_layout(p), 1, max_random)),
+        data("out1_lo", get_mem(get_single_element_layout(p), 0)),
+        data("out1_hi", get_mem(get_single_element_layout(p), 255)),
+        data("eltw_data", get_mem(layout(p.data_type, p.input_format, p.out_shape))),
+        data("scale2_data", get_mem(get_per_channel_layout(p), 1e-1f)),
+        data("in2_lo", get_mem(get_per_channel_layout(p), min_random, 0)),
+        data("in2_hi", get_mem(get_per_channel_layout(p), 1, max_random)),
+        data("out2_lo", get_mem(get_single_element_layout(p), -127)),
+        data("out2_hi", get_mem(get_single_element_layout(p), 127)),
+        permute("permute", "input", p.permute_order),
+        scale("scale1", "permute", "scale1_data"),
+        activation("actv1", "scale1", activation_func::relu),
+        quantize("quant1", "actv1", "in1_lo", "in1_hi", "out1_lo", "out1_hi", 256, data_types::f32),
+        eltwise("eltw", {"quant1", "eltw_data"}, eltwise_mode::sum, p.data_type),
+        scale("scale2", "eltw", "scale2_data"),
+        activation("actv2", "scale2", activation_func::relu),
+        quantize("quant2", "actv2", "in2_lo", "in2_hi", "out2_lo", "out2_hi", 255, data_types::i8),
+        reorder("out", "quant2", p.default_format, p.default_type)
+    );
+
+    tolerance = 1e-5f;
```

Comment:
I'd suggest removing `permute_quant_f32_eltwise_quant_i8` test completely and update this test to have only 1 quantize in the end of the topology (like `permute_scale_actv_eltw_scale_actv_quant_i8`) with 1.0f tolerance. 
Tests with 2 Quantize nodes fused are very sensitive to the input data and usually require tolerance ~2.0f to provide stable results. I don't think there's any difference between 1 and 2 fused quantize primitive from the Permute primitive perspective, it's more like jit generator testing which is covered by the other test cases.

Reply:
I changed tests to use only single quantization with 1.0f tolerance.